packages/feedback-service/src/resolver.ts

```diff
@@ -7,12 +7,14 @@ export const FeedbackResolver = {
   Query: {
     listFeedback(root: any, args: any, ctx: any) {
       return Feedback.find()
+        .populate('createdBy')
```

Comment:
This will no longer work. Did you test the APIs before committing?

Also, here you need to fetch the user details from User/Group Microservice using its API.
And add it to the feedback response and return it.

And the part about sending a single request, all the user data that is needed for the feedback response should be fetched using a single GraphQL query.
Check `getUsersBy` query in User/Group Microservice. (#281)